go.mod

```diff
@@ -4,7 +4,7 @@ go 1.19
 
 require (
 	github.com/google/go-cmp v0.5.9
-	github.com/hashicorp/terraform-plugin-go v0.16.0
+	github.com/hashicorp/terraform-plugin-go v0.16.1-0.20230627170012-6f0f2df12a0c
```

Comment:
Will this be updated to use changes in [tfprotov5+tfprotov6: Add DynamicValue type IsNull method](https://github.com/hashicorp/terraform-plugin-go/pull/306#top) once they're merged and released?

Reply:
Yep! Updating right now since that was released as v0.17.0